### DISCUSSION TITLE
请问如何实现以下箭头功能，在只有一边有节点的情况下，能指出和指入箭头

### DISCUSSION THREAD

**Discussion by Duanxy835** [antvis/X6#2158] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/33226907/171083623-4579712d-4faf-416e-8696-d51170251c01.png)
![image](https://user-images.githubusercontent.com/33226907/171083741-cb59fa58-2e86-4fcc-9864-225e33fcfd1c.png)

**Comment by NewByVector** [comment, marked as answer]
连线的 source 和 target 都可以为 Point, 比如：
```ts
graph.addEdge({
  source: [0, 0],
  target: [100, 100]
})
```
